blog_system/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.conf import settings
+from django.conf.urls.static import static
+from django.contrib import admin
+from django.urls import path
+from django.urls.conf import include
+from debug_toolbar.toolbar import debug_toolbar_urls
```

Comment:
The `debug_toolbar` module is not a part of the standard Django package and is typically used for debugging purposes during development. It's important to ensure that it is not included in production code. Consider setting up your project to include this module only in development settings.